### PR TITLE
add mask and trim flag

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -43,13 +43,75 @@ module.exports =
 /************************************************************************/
```
Comment (juliangruber) [inline dist/index.js:43]: This file should not be committed

File: action.yml [hunk 2]
```diff
@@ -8,6 +8,12 @@ inputs:
   path:
     description: 'File path'
     required: true
+  mask:
+    description: 'Hidden file content'
```
Comment (juliangruber) [inline action.yml:12]: It would be nice to say how the file content gets hidden

**Review by juliangruber — COMMENTED** [review]
Are you planning to reopen this? The changes looked good overall

**Comment by septs** [comment]
I wrote a new gh-action, https://github.com/NiceLabs/rclone-action

**Comment by juliangruber** [comment]
ok

**Pull request closed without merging**